Samples/Windowing/cpp-winui/SampleApp (Package)/SampleApp (Package).wapproj

```diff
@@ -66,7 +66,7 @@
   </ItemGroup>
   <ItemGroup>
     <PackageReference Include="Microsoft.Windows.SDK.BuildTools" Version="10.0.20348.19" />
-    <PackageReference Include="Microsoft.WindowsAppSDK" Version="1.0.0" />
+    <PackageReference Include="Microsoft.WindowsAppSDK" Version="1.1.0-preview2" />
```

Comment:
This should be "1.1.0" not targeting the preview2 release.
Note: this is perhaps not blocking for this PR, but needs to be updated once the 1.1.0 release is live.